imagick/magick_wand_image.go

```diff
@@ -765,6 +765,9 @@ func (mw *MagickWand) ExportImagePixels(x, y int, cols, rows uint,
 	pmap string, stype StorageType) (interface{}, error) {
 
 	maplen := (int(cols) - x) * (int(rows) - y) * len(pmap)
+	if maplen <= 0 {
+		return nil, errors.New("maplen <= 0 because the args are wrong")
```

Comment:
Can we fix this error up to be more meaningful to a caller? `maplen` is an implementation details, so it just tells you that "you did something wrong". 

You could first check that `len(pmap) != 0`, and report a specific error about it being a zero length string.
And then for the `maplen` part, you could report "Args x, y, cols, and rows produces an invalid region <= 0"